### DISCUSSION TITLE
Fall league FAQ answer is misleading

### DISCUSSION THREAD

**Discussion by nosnaws** [BattlesnakeOfficial/feedback#213] [Bug Reports]
Hello! I've been spending far too much time on the site waiting for the fall league to start and noticed some strange wording in the FAQ for this season.

### Summary
The fall league FAQ question "How spicy are the hazards?" seems to imply that hazard squares will not be immediately deadly. 

> How spicy are the hazards?
>For Fall League 2022, hazards do 100 damage when entered, which means any Battlesnake traveling through two hazard squares will be eliminated.
>The [settings in the data passed to your Battlesnake on every move request](https://docs.battlesnake.com/api/objects/ruleset-settings) will tell you how much health will be removed by moving onto a hazard. Your Battlesnake could strategically take a “dip in the sauce” and survive, but in Fall League 2022 it will be very risky.

The lines `which means any Battlesnake traveling through two hazard squares will be eliminated` and `Your Battlesnake could strategically take a “dip in the sauce” and survive, but in Fall League 2022 it will be very risky.` seem to contradict the [fall league guide](https://blog.battlesnake.com/fall-league-registration-is-open/). Which describes collisions with hazards as "fatal". 

Since the custom game maker and the fall league guide point to hazards being fatal, I'm assuming that the FAQ just needs an update.

**Comment by chris-bsnake** [comment]
Thanks for letting us know, i'll get that fixed today
